### PR TITLE
Dm 1237

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/secure/RapidBoard.jspa?rapidView=22&modal=detail&selectedIssue=DM-1237

## Description - what does this code do?
UX tweaks for practice show page

## Testing done - how did you test it/steps on how can another person can test it 
Added a test for visiting a commentator's profile page after clicking on their name next to their comment.

## Screenshots, Gifs, Videos from application (if applicable)
Screen Shot 2020-03-09 at 7 03 20 PM
Screen Shot 2020-03-09 at 6 58 25 PM
Screen Shot 2020-03-09 at 6 58 43 PM
Screen Shot 2020-03-09 at 6 58 55 PM
Screen Shot 2020-03-09 at 6 59 37 PM

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/DDUSxYiD1MaWNrGFmMjY8d8a/Diffusion-Marketplace-Designs?node-id=1745%3A27776

## Acceptance criteria
Go to spreadsheet and make sure you’re on the “Practice page” Tab

Make the changes and tweaks listed in the spreadsheet

Highlight them as BLUE(da ba di da ba dow) when you finish them 

There are 7 Practice Page bugs (Rows 3-12)

The rows highlighted in green are UI tasks, I’ve left them in there for FYI only

## Definition of done
- [X] Unit tests written (if applicable)
- [x] e2e/accessibility tests written (if applicable)
- [X] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/assets/stylesheets/commontator/comments.scss [hunk 1]
```diff
@@ -253,6 +256,108 @@ body {
...
+}
+
+.fa-flag:hover {
+  color: #E41D3D;
+}
+
+//#team-member:checked + .team-member-label::before, #verified-implementer:checked + .implementer-label::before {
+//  box-shadow: 0 0 0 2px #FFBE2E, inset 0 0 0 2px white !important;
+//  background-color: #FFBE2E !important;
+//}
```
Comment (ahay-agile6) [inline app/assets/stylesheets/commontator/comments.scss]: remove

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -51,6 +51,7 @@
 
 
     <title>Diffusion Marketplace</title>
+    <!--    <meta http-equiv="X-UA-Compatible" content="IE=edge">-->
```
Comment (ahay-agile6) [inline app/views/layouts/application.html.erb]: why'd we take this off?

Comment (ahay-agile6) [reply app/views/layouts/application.html.erb]: nevermind, we can remove this~

File: app/views/practices/_support_team.html.erb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <!-- "Team" section -->
 <section class="usa-section margin-y-0 p0">
     <div class="grid-row flex-align-center margin-bottom-3">
-      <h2 id="impact" class="font-sans-xl text-bold margin-0 flex-auto margin-right-205">Contact</h2>
-      <div class="height-2px border border-base-lightest flex-fill"></div>
+      <h2 id="contact" class="font-sans-xl text-bold margin-0 flex-auto margin-right-205 nav-header">Contact</h2>
```
[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/views/practices/_support_team.html.erb]: Which side-nav item has an id?

File: app/assets/stylesheets/dm_uswds/theme/_uswds-theme-custom-styles.scss [hunk 4]
```diff
@@ -115,9 +115,24 @@ i.e.
...
+  background-color: transparent !important;
   color: #FFFFFF !important;
-  box-shadow: inset 0 0 0 2px #FFFFFF !important;
+  box-shadow: none !important;
+  border: 2px solid #ffffff !important;
+}
+
+.usa-button--outline {
+  box-shadow: none !important;
+  border: 2px solid #005ea2 !important;
```
[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/assets/stylesheets/dm_uswds/theme/_uswds-theme-custom-styles.scss]: Got it!

File: app/assets/stylesheets/commontator/comments.scss [hunk 5]
```diff
@@ -141,27 +135,19 @@ h2 {
...
-// }
-
 .edit-comment-icon, .fa-trash-alt, .fa-trash-restore-alt {
   color: #71767A;
   text-decoration: none;
 }
 
 .edit-comment-icon:hover, .fa-trash-alt:hover, .fa-trash-restore-alt:hover {
-  color: #005EA2;
+  color: #565c65;
```
[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/assets/stylesheets/commontator/comments.scss]: Updating!

File: app/assets/stylesheets/commontator/comments.scss [hunk 6]
```diff
@@ -253,6 +237,103 @@ body {
   cursor: pointer;
 }
 
+.report-abuse-submit {
+    border: 2px solid #005ea2 !important;
+  &:hover {
+    border: 2px solid #1A4480 !important;
```
[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/assets/stylesheets/commontator/comments.scss]: Updating!

File: app/assets/stylesheets/commontator/comments.scss [hunk 7]
```diff
@@ -253,6 +237,103 @@ body {
...
+.hide-report-comment {
+  display: none !important;
+}
+
+.fa-flag:hover {
+  color: #E41D3D;
+}
+
+.usa-radio__input:focus + .team-member-label::before, .usa-radio__input:focus + .implementer-label::before {
+  outline: 0.25rem solid #FFBE2E !important;
```
[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/assets/stylesheets/commontator/comments.scss]: Updating!

File: app/assets/stylesheets/commontator/comments.scss [hunk 8]
```diff
@@ -253,6 +237,103 @@ body {
...
+}
+
+.more {
+  p {
+    padding-bottom: 4px;
+  }
+}
+
+.fa-user-circle {
+  color: #162e51 !important;
```
[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/assets/stylesheets/commontator/comments.scss]: Updating!

File: app/views/practices/_support_team.html.erb (hunk 3 above)
[the remark this answers is not in the document]
Comment (emyl3) [reply app/views/practices/_support_team.html.erb]: so in the `_show_authenticated.html.erb` file it looks like there is a `<section>` with the the same `id=contact`

[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/views/practices/_support_team.html.erb]: Removed. Thanks!

**Pull request merged**